child_compassion/views/project_reservation_view.xml

```diff
@@ -33,6 +33,7 @@
                                 <field name="hold_yield_rate" />
                                 <field name="is_reservation_auto_approved" />
                                 <field name="secondary_owner" />
+                                <button name="show_reserved_children" type="object" string="Show reserved children" />
```

Comment:
It's better to have a nice icon button at the upper right corner of the view. See how it's done for disaster alerts : 

https://github.com/CompassionCH/compassion-modules/blob/master/child_compassion/views/field_office_disaster_alert_view.xml#L20
http://fontawesome.io/icons/